scss/4-elements/_typography.scss

```diff
@@ -66,7 +69,7 @@
 h2,
 h3,
 h4 {
-  @include cads-adjacent-heading-margin($cads-spacing-6);
+  @include cads-adjacent-heading-margin($cads-spacing-7);
```

Comment:
Interested to know why H1 is not included here?

Reply:
This mixin adds a margin to the top of these heading elements when they appear after certain elements in a `.cads-prose` container.  The 'certain elements' I refer to are listed in the mixin:  

https://github.com/citizensadvice/design-system/blob/3e48703f9f6aea9bbb86f22122874c0063847792/scss/2-tools/_typography.scss#L24

It's responsible for the margin on the top of this heading element, for example: https://www.citizensadvice.org.uk/immigration/staying-in-the-uk-if-youre-from-the-eu#h-check-when-to-apply-for-the-eu-settlement-scheme

`h1`s will never have any of these elements above them so we don't need the top margin (currently).